### DISCUSSION TITLE
java.lang.ArrayIndexOutOfBoundsException

### DISCUSSION THREAD

**Discussion by DEKADENXIA** [dbeaver/dbeaver#11262] [DBeaver Development] [answered]
In previous versions you could insert massive registers with shift v control, but now I get this error: java.lang.ArrayIndexOutOfBoundsException

**Comment by LonwoLonwo** [comment]
Need more details.
What database are you using? DBeaver version?
Are you talking about Advanced past in Data Viewer?
If it is Advanced past - what type of columns is the data inserted into? How much data is inserted roughly?

**Comment by DEKADENXIA** [comment]
Until version 7.3.0 the records are allowed to be inserted, the database is informix 12, the number of records in version 7.3.0 I have inserted up to 1000, but in later versions only with two records it fails


________________________________
De: Anastasiya <notifications@github.com>
Enviado: martes, 9 de febrero de 2021 4:06 p. m.
Para: dbeaver/dbeaver <dbeaver@noreply.github.com>
Cc: DEKADENXIA <decadenxia@hotmail.es>; Author <author@noreply.github.com>
Asunto: Re: [dbeaver/dbeaver] java.lang.ArrayIndexOutOfBoundsException (#11262)


Need more details.
What database are you using? DBeaver version?
Are you talking about Advanced past in Data Viewer?
If it is Advanced past - what type of columns is the data inserted into? How much data is inserted roughly?

**Comment by LonwoLonwo** [comment]
Can you make a video of error with steps? I can't reproduce.

**Comment by LonwoLonwo** [comment, marked as answer]
Probably fixed in https://github.com/dbeaver/dbeaver/issues/11113
